src/content/pages/volunteers.mdx

```diff
@@ -3,26 +3,32 @@ title: Volunteers
 subtitle: We need your help to make EuroPython 2024 a success!
 ---
 
-# Volunteering@EuroPython
+# Volunteering @ EuroPython
```

Comment:
Why? Is this just a preference or is there something wrong on the page?

Reply:
Just a preference, not a strong one though. We can keep it as is

Reply:
It seems we have no precendence for 2024 ... - So let's go with your version for now.